build.py

```diff
@@ -0,0 +1,396 @@
+from __future__ import annotations
+
+import argparse
+import configparser
+import contextlib
+import functools
+import itertools
+import json
+import os.path
+import platform
+import shutil
+import subprocess
+import sys
+import tempfile
+import urllib.error
+import urllib.parse
+import urllib.request
+from typing import Callable
+from typing import ContextManager
+from typing import Generator
+from typing import Mapping
+from typing import NamedTuple
+
+from packaging.tags import compatible_tags
+from packaging.tags import cpython_tags
+from packaging.tags import platform_tags
+from packaging.tags import Tag
+from packaging.utils import parse_wheel_filename
+from packaging.version import Version
+
+PYTHONS = ((3, 8), (3, 9), (3, 10))
```

Comment:
Can we just build for 3.8? If this is being done proactively, I would see us just upgrading directly to 3.11 in the future.